### PR TITLE
Handle epel installs/removals differently

### PR DESCRIPTION
**Pull request by lpramuk**
- the logic is if epel was enabled before an action leave it also enabled afterwards (useful for nightly)
- otherwise if epel has been enabled then remove it afterwards (usual downstream)

### CODE REVIEW

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -237,10 +237,13 @@ def setup_avahi_discovery():
        by 'ping vm.local' run at Satellite
     """
     os_version = distro_info()[1]
-    run('yum -y install https://dl.fedoraproject.org/pub/epel/'
-        'epel-release-latest-{0}.noarch.rpm'.format(os_version))
+    epel_was_missing = not run('rpm -qa epel-release')
```
Comment (lpramuk) [inline automation_tools/__init__.py]: Note: rpm -qa epel-release
$?=0 always

Comment (lpramuk) [reply automation_tools/__init__.py]: so no need for warn_only=True

Comment (kbidarkar) [reply automation_tools/__init__.py]: How about we do it this way?
```
epel_missing = run('rpm -qa epel-release')
if "epel-release" not in epel_missing:
     run('yum -y install https://dl.fedoraproject.org/pub/epel/'
           'epel-release-latest-{0}.noarch.rpm'.format(os_version))
```

Comment (lpramuk) [reply automation_tools/__init__.py]: I've just realized that fabric supports ```return_code```
Using ```return_code``` it would be most precise.

**Comment by lpramuk** [comment]
Comments resolved.

**Pull request merged**